tests/cpp17/generated_cpp17/optional_scalars_generated.h

```diff
@@ -4,6 +4,8 @@
 #ifndef FLATBUFFERS_GENERATED_OPTIONALSCALARS_OPTIONAL_SCALARS_H_
 #define FLATBUFFERS_GENERATED_OPTIONALSCALARS_OPTIONAL_SCALARS_H_
 
+#include <tuple>
```

Comment:
Is `<tuple>` really needed?

Reply:
Ah you are right it is no longer need (that's good, because it is a heavy header template-wise).  So I have removed it now.